go/teams/service_helper.go

```diff
@@ -860,13 +870,10 @@ func TeamAcceptInviteOrRequestAccess(ctx context.Context, g *libkb.GlobalContext
 	maybeSeitan, keepSecret := ParseSeitanTokenFromPaste(tokenOrName)
 	if keepSecret {
 		g.Log.CDebugf(ctx, "found seitan-ish token")
-		seitan, err := GenerateIKeyFromString(maybeSeitan)
+		wasSeitan, err := ParseAndAcceptSeitanToken(ctx, g, maybeSeitan)
 		if err != nil {
-			return keybase1.TeamAcceptOrRequestResult{}, err
+			return keybase1.TeamAcceptOrRequestResult{WasSeitan: wasSeitan}, err
```

Comment:
The return should be unconditional. Otherwise when `errr == nil` it'll fall through to email-style

Reply:
Yup, will fix.